src/assets/components/ViewArticle.jsx

```diff
@@ -1,26 +1,56 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../../utils";
+import { getArticleById, getCommentByArticleId } from "../../utils";
 
 export default function ViewArticle() {
     const {article_id} = useParams()
     const [article, setArticle] = useState({})
+    const [comments, setComments] = useState([])
 
     useEffect(() => {
         getArticleById(article_id).then((response) => {
             setArticle(response.article)
         })
     }, [article_id])
 
+    useEffect(() => {
+        getCommentByArticleId(article_id).then((response) => {
+            setComments(response.comments)
+        })
+    })
```

Comment:
Please include the dependency array here, as currently you have an infinite loop running and sending requests to your API continuously.